packages/api/db/migration/20230920151515_add_guernsey_aland_to_countries.js

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022, 2023 LiteFarm.org
```

Comment:
You only need the 2023 (for future reference, not a required change)

Reply:
fixed it